docs/usage/basic_usage.rst

```diff
@@ -3,7 +3,7 @@ Basic usage
 
 The general workflow to execute Python code that is loaded within a Python program is:
 
-.. testcode::
+.. code-block:: python
```

Comment:
please do not change ``.. testcode::`` derictives to ``code-block`` derictives, as this changes also the behaviour.

code-block only highlights the code,
testcode actualy allows to perform tests on that code

see http://www.sphinx-doc.org/en/stable/ext/doctest.html

Reply:
done